pipeline/src/node_properties/model_inference_properties.rs

```diff
@@ -48,5 +48,16 @@ pub struct ModelInferenceRuntime {
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 pub struct ClassInferenceProperties {
     /// Minimum inference threshold, should be set on the [0.0, 1.0] interval.
-    pub min_inference_threshold: f32,
+    pub min_inference_threshold: Option<f32>,
```

Comment:
There's no mention in the CH story or the commit message why this existing property is being made optional.

Reply:
When we had a single inference property, it was possible to figure if it was enabled or disabled just based on the `class_properties`
```pub class_properties: Option<BTreeMap<String, ClassInferenceProperties>>,```

but now we can have the case where user sets the min/max bbox dimensions, but does not tweak the `min_inference_threshold`

Reply:
Yeah, this change should be in a separate commit IMO.

Reply:
alternative would be remove the options and console fill the not set properties with a negative value in order we can discard it later on lumeod.

Reply:
> Yeah, this change should be in a separate commit IMO.

OK, I will extract it to a separate commit and add the reasoning on commit log as soon we settle the changes on the main commit.